src/main/proto/netflix/titus/titus_base.proto

```diff
@@ -301,6 +301,14 @@ message SignedAddressAllocation {
     /// (Required) The IP address and metadata that was allocated.
     AddressAllocation addressAllocation = 1;
 
-    // (Required) This is an x509v3 signed version of the address allocation.
-    bytes signedAddressAllocation = 2;
```

Comment:
CC'ing @tbak to see if there's a style preference for leaving this `= 2` field or just having it missing.

Reply:
If the IDL mode is not yet used in prod, we should drop it.

Reply:
It's not in prod yet, only in test main cells.

Reply:
As it was deployed, lets leave a comment that this number should not be reused.